src/quizy/bp_quizy.py

```diff
@@ -0,0 +1,47 @@
+from flask import Blueprint, redirect, render_template, request, session, url_for
+from flask_user import current_user, login_required
+
+from quizy.app_data import get_quizy_data
+from quizy.questions import generate_questions
+
+
+bp_quizy = Blueprint('quizy', __name__, url_prefix='/quizy', template_folder='templates/quizy')
+
+
+@bp_quizy.route('/choose')
+@login_required
+def choose_page():
+    return render_template('choose.html')
+
+
+@bp_quizy.route('/create', methods=['POST'])
+@login_required
+def create_page():
+    difficulty = request.form.get('difficulty')
+    try:
+        questions = generate_questions(difficulty=difficulty)
+    except (ConnectionError, ValueError):
```

Comment:
Tutaj może troszkę dziwne (z punktu widzenia User Experience) jest to zachowanie, że w przypadku problemów z połączeniem przekierowujesz po prostu do `choose_page`.